docs/configuration/configuration.md

```diff
@@ -1294,6 +1294,11 @@ url: <string>
 write_relabel_configs:
   [ - <relabel_config> ... ]
 
+# Name of the remote write config, useful for users who have multiple remote writes configured.
+# The name will be used in structured logging and as a label in remote write metrics to help users identify
+# a unique remote write queue.
```

Comment:
do we check for uniqueness?

Reply:
Yes, uniqueness is required as part of parsing the configuration.